dhall/src/Dhall/Main.hs

```diff
@@ -161,7 +161,7 @@ data Mode
           }
     | Encode { file :: Input, json :: Bool }
     | Decode { file :: Input, json :: Bool, quiet :: Bool }
-    | Text { file :: Input }
+    | Text { file :: Input, outputFile :: Maybe FilePath }
```

Comment:
Instead of using `Maybe FilePath` you could use `Output` that is defined on `Dhall.Util`. There are some other flags that use that for this purpose.

Check here for instance:
https://github.com/dhall-lang/dhall-haskell/blob/760d98536d3c7e6a5cbf26d9c7a900758add91b8/dhall/src/Dhall/Main.hs#L339-L348

Reply:
`Output` is a homomorphism for `Maybe FilePath`, but it's better to use that for consistency on this specific purpose though

Reply:
Indeed, thanks!